string dtype
InformSlot[] slots
float64 d_type_probability
string[] requested_slots